gengo/helpers.go

```diff
@@ -0,0 +1,40 @@
+package gengo
+
+import (
+	"specgen/gen"
+	"strings"
+)
+
+func generateHelperFunctions(packageName string, path string) *gen.TextFile {
```

Comment:
May be we need to get better name instead of "helpers" - not sure yet what the name would be